### PR TITLE
Add support for Github organisation level runners

### PR DESCRIPTION
**Pull request by JamesMowatt**
This PR adds support for organisation level runner registration in Github, you are able to register a runner that can be used by multiple repositories.

### CODE REVIEW

File: src/github.js [hunk 1]
```diff
@@ -62,14 +63,26 @@ const comment = async (opts) => {
...
 const get_runner_token = async () => {
-  const {
-    data: { token }
-  } = await octokit.actions.createRegistrationTokenForRepo({
-    owner,
-    repo
-  });
-
-  return token;
+  if (typeof owner !== 'undefined' && typeof repo !== 'undefined') {
```
Comment (DavidGOrtega) [inline src/github.js:66]: Lets remove this if putting the block below the organization code 

```js
if (typeof owner !== 'undefined' && typeof repo === 'undefined') {
//do
}


```

Comment (DavidGOrtega) [inline src/github.js:66]: typeof owner !== 'undefined' does not have to be checked since its a condition of both

**Review by DavidGOrtega — CHANGES_REQUESTED** [review]
Lets remove this if moving this block after the organization check if block

```js
if (/*is organization */) {
//do
}




```

**Comment by DavidGOrtega** [comment]
@JamesMowatt The idea is to pass  ```--RUNNER_REPO="https://github.com/myorg"``` ?

**Comment by DavidGOrtega** [comment]
Looks great! Just only minor issues @JamesMowatt 

Something missing is Gitlab. Same functionality has to be offered. 
Would be possible for you to do it? If not no worries.

**Comment by JamesMowatt** [comment]
> @JamesMowatt, thanks for the help! 
> 
> The idea is to pass `--RUNNER_REPO="https://github.com/myorg"` ?

Yes, you would only pass the organisation.



> Looks great! Just only minor issues @JamesMowatt
> 
> Something missing is Gitlab. Same functionality has to be offered.
> Would be possible for you to do it? If not no worries.

I will look to raise the Gitlab work as a separate PR if that is ok?

> Lets remove this if moving this block after the organization check if block
> 
> ```js
> if (/*is organization */) {
> //do
> }
> ```

I have applied these requested changes.

**Comment by DavidGOrtega** [comment]
Looks perfect!

**Pull request merged**